tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -257,7 +260,7 @@ public function test_blocking_dependent_with_delayed_dependency( $strategy ) {
 		wp_enqueue_script( 'main-script-a3', '/main-script-a3.js', array(), null, compact( 'strategy' ) );
 		wp_enqueue_script( 'dependent-script-a3', '/dependent-script-a3.js', array( 'main-script-a3' ), null );
 		$output   = get_echo( 'wp_print_scripts' );
-		$expected = "<script type='text/javascript' src='/main-script-a3.js' id='main-script-a3-js' data-wp-strategy='{$strategy}'></script>";
+		$expected = str_replace( "'", '"', "<script type='text/javascript' src='/main-script-a3.js' id='main-script-a3-js' data-wp-strategy='{$strategy}'></script>" );
```

Comment:
should this be a `assertEqualMarkup()` scenario?
if not, the HEREDOC might make it less quirky

```php
	$expected = <<<HTML
<script type="text/javascript" src="/main-script-a3.js" id="main-script-a3-js" data-wp-strategy="{$strategy}"></script>
HTML;
```

the initial and terminal newlines are stripped automatically, so this is a _trimmed_ string.

Reply:
Ah, well, it's actually checking for only one of the script tags of the two. But I can just add the other script so it is checking for both.

Reply:
See https://github.com/WordPress/wordpress-develop/pull/5301/commits/9e0c1f5db3343528b92deb99aaf730516f3e1637 as part of #5301